src/__tests__/test_helpers/asyncHelper.js

```diff
@@ -10,17 +10,17 @@
 // We need this until Jest finishes upgrading to Jasmine 2.0
 module.exports = function asyncHelper(fn) {
   let finished = false;
-  const done = function() {
+  const done = function () {
```

Comment:
I think we had that Lint discussion already, but the space between `function` and brackets is what we want?

Reply:
Yeah, technically you need a space before the name of the function. The name here is empty so the space should remain.